anchore_engine/twisted.py

```diff
@@ -269,7 +270,7 @@ def _build_api_service(self):
         root = rewrite.RewriterResource(self.root_resource, self._default_version_rewrite)
 
         # Build the main site server
-        site = server.Site(root)
+        site = server.Site(root, logPath=b'/var/log/anchore/access.log')
```

Comment:
woah, this needs to be bytes?

Reply:
What's the purpose of this entry? Does this now split the output for access and info/debug logs?

Reply:
@zhill yes, it separates those logs out so that it doesn't mix into the normal output of the logs. Twisted wasn't capturing these in it's normal Log Observers so this is the only way to not screw up the JSON logging piece. I could change this logic to only split it out if JSON logging is enabled, if you prefer?

Reply:
Ack. I'd like a configuration option for this to allow the user to specify if they want a unified log stream or a split out model.

Reply:
@zhill I think my assumption above was incorrect. Access logging doesn't seem to mess up the JSON logs (May have been a relic of some other local issues I had during development). I noticed that enabling/disabling access logging wasn't even working so I removed that for now.